client/src/components/shared/search.js

```diff
@@ -22,18 +23,24 @@ class SearchComponent extends Component{
      * CHECKS TARGET VALUE AND SETS PLACEHOLDER AND SEARCH TYPE STATE
      */
     setPlaceholder(event){
+        this.setState({
+            inputValue: []
+        });
         let target = event.target.value;
         if (target === 'members') {
-            this.setState({
-                placeholderText: 'Enter a member name..',
-                searchType: target,
-                searchResults: InputSearchService.fetchSearch(target)
-            })
+            axios.get('http://localhost:3004/member-list')
+                .then(result => {
+                    this.setState({
+                        searchType: target,
+                        placeholderText: 'Enter a member name...',
```

Comment:
This server call should be in the getRequest file of the actions folder then the response data should be put in the redux store. You want to set up a time to meet, and I can show you the flow?